File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/emsdi/d/EmsDiSrvMmDTO.java
package iih.ci.ord.emsdi.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 医疗单di项目物品DTO DTO数据 
 * 
 */
public class EmsDiSrvMmDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 医疗单di服务物品主键
	 * @return String
	 */
	public String getId_emsdisrvmm() {
		return ((String) getAttrVal("Id_emsdisrvmm"));
	}	
	/**
	 * 医疗单di服务物品主键
	 * @param Id_emsdisrvmm
	 */
	public void setId_emsdisrvmm(String Id_emsdisrvmm) {
		setAttrVal("Id_emsdisrvmm", Id_emsdisrvmm);
	}
	/**
	 * 医疗单di主键
	 * @return String
	 */
	public String getId_emsdi() {
		return ((String) getAttrVal("Id_emsdi"));
	}	
	/**
	 * 医疗单di主键
	 * @param Id_emsdi
	 */
	public void setId_emsdi(String Id_emsdi) {
		setAttrVal("Id_emsdi", Id_emsdi);
	}
	/**
	 * 医疗单di服务主键
	 * @return String
	 */
	public String getId_emsdisrv() {
		return ((String) getAttrVal("Id_emsdisrv"));
	}	
	/**
	 * 医疗单di服务主键
	 * @param Id_emsdisrv
	 */
	public void setId_emsdisrv(String Id_emsdisrv) {
		setAttrVal("Id_emsdisrv", Id_emsdisrv);
	}
	/**
	 * 服务id
	 * @return String
	 */
	public String getId_srv() {
		return ((String) getAttrVal("Id_srv"));
	}	
	/**
	 * 服务id
	 * @param Id_srv
	 */
	public void setId_srv(String Id_srv) {
		setAttrVal("Id_srv", Id_srv);
	}
	/**
	 * 物品
	 * @return String
	 */
	public String getId_mm() {
		return ((String) getAttrVal("Id_mm"));
	}	
	/**
	 * 物品
	 * @param Id_mm
	 */
	public void setId_mm(String Id_mm) {
		setAttrVal("Id_mm", Id_mm);
	}
	/**
	 * 药品编码
	 * @return String
	 */
	public String getCode_mm() {
		return ((String) getAttrVal("Code_mm"));
	}	
	/**
	 * 药品编码
	 * @param Code_mm
	 */
	public void setCode_mm(String Code_mm) {
		setAttrVal("Code_mm", Code_mm);
	}
	/**
	 * 物品名称
	 * @return String
	 */
	public String getName_mm() {
		return ((String) getAttrVal("Name_mm"));
	}	
	/**
	 * 物品名称
	 * @param Name_mm
	 */
	public void setName_mm(String Name_mm) {
		setAttrVal("Name_mm", Name_mm);
	}
	/**
	 * 规格
	 * @return String
	 */
	public String getSpec_mm() {
		return ((String) getAttrVal("Spec_mm"));
	}	
	/**
	 * 规格
	 * @param Spec_mm
	 */
	public void setSpec_mm(String Spec_mm) {
		setAttrVal("Spec_mm", Spec_mm);
	}
	/**
	 * 物品类型
	 * @return String
	 */
	public String getId_mmtp() {
		return ((String) getAttrVal("Id_mmtp"));
	}	
	/**
	 * 物品类型
	 * @param Id_mmtp
	 */
	public void setId_mmtp(String Id_mmtp) {
		setAttrVal("Id_mmtp", Id_mmtp);
	}
	/**
	 * 物品类型编码
	 * @return String
	 */
	public String getSd_mmtp() {
		return ((String) getAttrVal("Sd_mmtp"));
	}	
	/**
	 * 物品类型编码
	 * @param Sd_mmtp
	 */
	public void setSd_mmtp(String Sd_mmtp) {
		setAttrVal("Sd_mmtp", Sd_mmtp);
	}
	/**
	 * 物品类型名称
	 * @return String
	 */
	public String getName_mmtp() {
		return ((String) getAttrVal("Name_mmtp"));
	}	
	/**
	 * 物品类型名称
	 * @param Name_mmtp
	 */
	public void setName_mmtp(String Name_mmtp) {
		setAttrVal("Name_mmtp", Name_mmtp);
	}
	/**
	 * 厂家
	 * @return String
	 */
	public String getId_sup() {
		return ((String) getAttrVal("Id_sup"));
	}	
	/**
	 * 厂家
	 * @param Id_sup
	 */
	public void setId_sup(String Id_sup) {
		setAttrVal("Id_sup", Id_sup);
	}
	/**
	 * 厂家名称
	 * @return String
	 */
	public String getName_sup() {
		return ((String) getAttrVal("Name_sup"));
	}	
	/**
	 * 厂家名称
	 * @param Name_sup
	 */
	public void setName_sup(String Name_sup) {
		setAttrVal("Name_sup", Name_sup);
	}
	/**
	 * 取整方式
	 * @return String
	 */
	public String getSd_roundmd() {
		return ((String) getAttrVal("Sd_roundmd"));
	}	
	/**
	 * 取整方式
	 * @param Sd_roundmd
	 */
	public void setSd_roundmd(String Sd_roundmd) {
		setAttrVal("Sd_roundmd", Sd_roundmd);
	}
	/**
	 * 基本单位
	 * @return String
	 */
	public String getId_unit_base() {
		return ((String) getAttrVal("Id_unit_base"));
	}	
	/**
	 * 基本单位
	 * @param Id_unit_base
	 */
	public void setId_unit_base(String Id_unit_base) {
		setAttrVal("Id_unit_base", Id_unit_base);
	}
	/**
	 * 名称_基本单位
	 * @return String
	 */
	public String getName_unit_base() {
		return ((String) getAttrVal("Name_unit_base"));
	}	
	/**
	 * 名称_基本单位
	 * @param Name_unit_base
	 */
	public void setName_unit_base(String Name_unit_base) {
		setAttrVal("Name_unit_base", Name_unit_base);
	}
	/**
	 * 系数_医学基本
	 * @return FDouble
	 */
	public FDouble getFactor_mb() {
		return ((FDouble) getAttrVal("Factor_mb"));
	}	
	/**
	 * 系数_医学基本
	 * @param Factor_mb
	 */
	public void setFactor_mb(FDouble Factor_mb) {
		setAttrVal("Factor_mb", Factor_mb);
	}
	/**
	 * 单次数量分子
	 * @return FDouble
	 */
	public FDouble getQuan_num_base() {
		return ((FDouble) getAttrVal("Quan_num_base"));
	}	
	/**
	 * 单次数量分子
	 * @param Quan_num_base
	 */
	public void setQuan_num_base(FDouble Quan_num_base) {
		setAttrVal("Quan_num_base", Quan_num_base);
	}
	/**
	 * 单次数量分母
	 * @return String
	 */
	public String getQuan_den_base() {
		return ((String) getAttrVal("Quan_den_base"));
	}	
	/**
	 * 单次数量分母
	 * @param Quan_den_base
	 */
	public void setQuan_den_base(String Quan_den_base) {
		setAttrVal("Quan_den_base", Quan_den_base);
	}
	/**
	 * 总量单位
	 * @return String
	 */
	public String getId_pgku_cur() {
		return ((String) getAttrVal("Id_pgku_cur"));
	}	
	/**
	 * 总量单位
	 * @param Id_pgku_cur
	 */
	public void setId_pgku_cur(String Id_pgku_cur) {
		setAttrVal("Id_pgku_cur", Id_pgku_cur);
	}
	/**
	 * 名称_总量单位
	 * @return String
	 */
	public String getName_pgku_cur() {
		return ((String) getAttrVal("Name_pgku_cur"));
	}	
	/**
	 * 名称_总量单位
	 * @param Name_pgku_cur
	 */
	public void setName_pgku_cur(String Name_pgku_cur) {
		setAttrVal("Name_pgku_cur", Name_pgku_cur);
	}
	/**
	 * 系数_当前基本
	 * @return FDouble
	 */
	public FDouble getFactor_cb() {
		return ((FDouble) getAttrVal("Factor_cb"));
	}	
	/**
	 * 系数_当前基本
	 * @param Factor_cb
	 */
	public void setFactor_cb(FDouble Factor_cb) {
		setAttrVal("Factor_cb", Factor_cb);
	}
	/**
	 * 参考价格
	 * @return FDouble
	 */
	public FDouble getPrice() {
		return ((FDouble) getAttrVal("Price"));
	}	
	/**
	 * 参考价格
	 * @param Price
	 */
	public void setPrice(FDouble Price) {
		setAttrVal("Price", Price);
	}
	/**
	 * 总量
	 * @return FDouble
	 */
	public FDouble getQuan_cur() {
		return ((FDouble) getAttrVal("Quan_cur"));
	}	
	/**
	 * 总量
	 * @param Quan_cur
	 */
	public void setQuan_cur(FDouble Quan_cur) {
		setAttrVal("Quan_cur", Quan_cur);
	}
	/**
	 * 金额
	 * @return FDouble
	 */
	public FDouble getAmt_cur() {
		return ((FDouble) getAttrVal("Amt_cur"));
	}	
	/**
	 * 金额
	 * @param Amt_cur
	 */
	public void setAmt_cur(FDouble Amt_cur) {
		setAttrVal("Amt_cur", Amt_cur);
	}
	/**
	 * 可用库存
	 * @return FDouble
	 */
	public FDouble getQuan_stock() {
		return ((FDouble) getAttrVal("Quan_stock"));
	}	
	/**
	 * 可用库存
	 * @param Quan_stock
	 */
	public void setQuan_stock(FDouble Quan_stock) {
		setAttrVal("Quan_stock", Quan_stock);
	}
	/**
	 * 药品剂型
	 * @return String
	 */
	public String getId_dosage() {
		return ((String) getAttrVal("Id_dosage"));
	}	
	/**
	 * 药品剂型
	 * @param Id_dosage
	 */
	public void setId_dosage(String Id_dosage) {
		setAttrVal("Id_dosage", Id_dosage);
	}
	/**
	 * 药品剂型编码
	 * @return String
	 */
	public String getSd_dosage() {
		return ((String) getAttrVal("Sd_dosage"));
	}	
	/**
	 * 药品剂型编码
	 * @param Sd_dosage
	 */
	public void setSd_dosage(String Sd_dosage) {
		setAttrVal("Sd_dosage", Sd_dosage);
	}
	/**
	 * 药理分类
	 * @return String
	 */
	public String getId_pharm() {
		return ((String) getAttrVal("Id_pharm"));
	}	
	/**
	 * 药理分类
	 * @param Id_pharm
	 */
	public void setId_pharm(String Id_pharm) {
		setAttrVal("Id_pharm", Id_pharm);
	}
	/**
	 * 药理分类编码
	 * @return String
	 */
	public String getSd_pharm() {
		return ((String) getAttrVal("Sd_pharm"));
	}	
	/**
	 * 药理分类编码
	 * @param Sd_pharm
	 */
	public void setSd_pharm(String Sd_pharm) {
		setAttrVal("Sd_pharm", Sd_pharm);
	}
	/**
	 * 毒麻分类
	 * @return String
	 */
	public String getId_pois() {
		return ((String) getAttrVal("Id_pois"));
	}	
	/**
	 * 毒麻分类
	 * @param Id_pois
	 */
	public void setId_pois(String Id_pois) {
		setAttrVal("Id_pois", Id_pois);
	}
	/**
	 * 毒麻分类编码
	 * @return String
	 */
	public String getSd_pois() {
		return ((String) getAttrVal("Sd_pois"));
	}	
	/**
	 * 毒麻分类编码
	 * @param Sd_pois
	 */
	public void setSd_pois(String Sd_pois) {
		setAttrVal("Sd_pois", Sd_pois);
	}
	/**
	 * 抗菌药分类
	 * @return String
	 */
	public String getId_anti() {
		return ((String) getAttrVal("Id_anti"));
	}	
	/**
	 * 抗菌药分类
	 * @param Id_anti
	 */
	public void setId_anti(String Id_anti) {
		setAttrVal("Id_anti", Id_anti);
	}
	/**
	 * 抗菌药分类编码
	 * @return String
	 */
	public String getSd_anti() {
		return ((String) getAttrVal("Sd_anti"));
	}	
	/**
	 * 抗菌药分类编码
	 * @param Sd_anti
	 */
	public void setSd_anti(String Sd_anti) {
		setAttrVal("Sd_anti", Sd_anti);
	}
	/**
	 * 价值分类
	 * @return String
	 */
	public String getId_val() {
		return ((String) getAttrVal("Id_val"));
	}	
	/**
	 * 价值分类
	 * @param Id_val
	 */
	public void setId_val(String Id_val) {
		setAttrVal("Id_val", Id_val);
	}
	/**
	 * 价值分类编码
	 * @return String
	 */
	public String getSd_val() {
		return ((String) getAttrVal("Sd_val"));
	}	
	/**
	 * 价值分类编码
	 * @param Sd_val
	 */
	public void setSd_val(String Sd_val) {
		setAttrVal("Sd_val", Sd_val);
	}
	/**
	 * OTC标识
	 * @return FBoolean
	 */
	public FBoolean getFg_otc() {
		return ((FBoolean) getAttrVal("Fg_otc"));
	}	
	/**
	 * OTC标识
	 * @param Fg_otc
	 */
	public void setFg_otc(FBoolean Fg_otc) {
		setAttrVal("Fg_otc", Fg_otc);
	}
	/**
	 * 皮试标识
	 * @return FBoolean
	 */
	public FBoolean getFg_skintest() {
		return ((FBoolean) getAttrVal("Fg_skintest"));
	}	
	/**
	 * 皮试标识
	 * @param Fg_skintest
	 */
	public void setFg_skintest(FBoolean Fg_skintest) {
		setAttrVal("Fg_skintest", Fg_skintest);
	}
	/**
	 * 对应皮试服务
	 * @return String
	 */
	public String getId_srvskin() {
		return ((String) getAttrVal("Id_srvskin"));
	}	
	/**
	 * 对应皮试服务
	 * @param Id_srvskin
	 */
	public void setId_srvskin(String Id_srvskin) {
		setAttrVal("Id_srvskin", Id_srvskin);
	}
	/**
	 * 关联信息Map键值串
	 * @return String
	 */
	public String getMapkeys() {
		return ((String) getAttrVal("Mapkeys"));
	}	
	/**
	 * 关联信息Map键值串
	 * @param Mapkeys
	 */
	public void setMapkeys(String Mapkeys) {
		setAttrVal("Mapkeys", Mapkeys);
	}
	/**
	 * 服务物品关联信息MAP
	 * @return FMap
	 */
	public FMap getMapinfo() {
		return ((FMap) getAttrVal("Mapinfo"));
	}	
	/**
	 * 服务物品关联信息MAP
	 * @param Mapinfo
	 */
	public void setMapinfo(FMap Mapinfo) {
		setAttrVal("Mapinfo", Mapinfo);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/emsdi/d/EmsDiSrvMmDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.emsdi.d
{
    /// <summary>
    /// EmsDiSrvMmDTO 的摘要说明。
    /// </summary>
    public class EmsDiSrvMmDTO : BaseDTO {

        public EmsDiSrvMmDTO() {
 
        }

        /// <summary>
        /// 医疗单di服务物品主键
        /// </summary>
		public string Id_emsdisrvmm {
            get { return getAttrVal<string>("Id_emsdisrvmm",null); }
            set { setAttrVal<string>("Id_emsdisrvmm", value); }
        }

        /// <summary>
        /// 医疗单di主键
        /// </summary>
		public string Id_emsdi {
            get { return getAttrVal<string>("Id_emsdi",null); }
            set { setAttrVal<string>("Id_emsdi", value); }
        }

        /// <summary>
        /// 医疗单di服务主键
        /// </summary>
		public string Id_emsdisrv {
            get { return getAttrVal<string>("Id_emsdisrv",null); }
            set { setAttrVal<string>("Id_emsdisrv", value); }
        }

        /// <summary>
        /// 服务id
        /// </summary>
		public string Id_srv {
            get { return getAttrVal<string>("Id_srv",null); }
            set { setAttrVal<string>("Id_srv", value); }
        }

        /// <summary>
        /// 物品
        /// </summary>
		public string Id_mm {
            get { return getAttrVal<string>("Id_mm",null); }
            set { setAttrVal<string>("Id_mm", value); }
        }

        /// <summary>
        /// 药品编码
        /// </summary>
		public string Code_mm {
            get { return getAttrVal<string>("Code_mm",null); }
            set { setAttrVal<string>("Code_mm", value); }
        }

        /// <summary>
        /// 物品名称
        /// </summary>
		public string Name_mm {
            get { return getAttrVal<string>("Name_mm",null); }
            set { setAttrVal<string>("Name_mm", value); }
        }

        /// <summary>
        /// 规格
        /// </summary>
		public string Spec_mm {
            get { return getAttrVal<string>("Spec_mm",null); }
            set { setAttrVal<string>("Spec_mm", value); }
        }

        /// <summary>
        /// 物品类型
        /// </summary>
		public string Id_mmtp {
            get { return getAttrVal<string>("Id_mmtp",null); }
            set { setAttrVal<string>("Id_mmtp", value); }
        }

        /// <summary>
        /// 物品类型编码
        /// </summary>
		public string Sd_mmtp {
            get { return getAttrVal<string>("Sd_mmtp",null); }
            set { setAttrVal<string>("Sd_mmtp", value); }
        }

        /// <summary>
        /// 物品类型名称
        /// </summary>
		public string Name_mmtp {
            get { return getAttrVal<string>("Name_mmtp",null); }
            set { setAttrVal<string>("Name_mmtp", value); }
        }

        /// <summary>
        /// 厂家
        /// </summary>
		public string Id_sup {
            get { return getAttrVal<string>("Id_sup",null); }
            set { setAttrVal<string>("Id_sup", value); }
        }

        /// <summary>
        /// 厂家名称
        /// </summary>
		public string Name_sup {
            get { return getAttrVal<string>("Name_sup",null); }
            set { setAttrVal<string>("Name_sup", value); }
        }

        /// <summary>
        /// 取整方式
        /// </summary>
		public string Sd_roundmd {
            get { return getAttrVal<string>("Sd_roundmd",null); }
            set { setAttrVal<string>("Sd_roundmd", value); }
        }

        /// <summary>
        /// 基本单位
        /// </summary>
		public string Id_unit_base {
            get { return getAttrVal<string>("Id_unit_base",null); }
            set { setAttrVal<string>("Id_unit_base", value); }
        }

        /// <summary>
        /// 名称_基本单位
        /// </summary>
		public string Name_unit_base {
            get { return getAttrVal<string>("Name_unit_base",null); }
            set { setAttrVal<string>("Name_unit_base", value); }
        }

        /// <summary>
        /// 系数_医学基本
        /// </summary>
		public double? Factor_mb {
            get { return getAttrVal<FDouble>("Factor_mb",null); }
            set { setAttrVal<FDouble>("Factor_mb", value); }
        }

        /// <summary>
        /// 单次数量分子
        /// </summary>
		public double? Quan_num_base {
            get { return getAttrVal<FDouble>("Quan_num_base",null); }
            set { setAttrVal<FDouble>("Quan_num_base", value); }
        }

        /// <summary>
        /// 单次数量分母
        /// </summary>
		public string Quan_den_base {
            get { return getAttrVal<string>("Quan_den_base",null); }
            set { setAttrVal<string>("Quan_den_base", value); }
        }

        /// <summary>
        /// 总量单位
        /// </summary>
		public string Id_pgku_cur {
            get { return getAttrVal<string>("Id_pgku_cur",null); }
            set { setAttrVal<string>("Id_pgku_cur", value); }
        }

        /// <summary>
        /// 名称_总量单位
        /// </summary>
		public string Name_pgku_cur {
            get { return getAttrVal<string>("Name_pgku_cur",null); }
            set { setAttrVal<string>("Name_pgku_cur", value); }
        }

        /// <summary>
        /// 系数_当前基本
        /// </summary>
		public double? Factor_cb {
            get { return getAttrVal<FDouble>("Factor_cb",null); }
            set { setAttrVal<FDouble>("Factor_cb", value); }
        }

        /// <summary>
        /// 参考价格
        /// </summary>
		public double? Price {
            get { return getAttrVal<FDouble>("Price",null); }
            set { setAttrVal<FDouble>("Price", value); }
        }

        /// <summary>
        /// 总量
        /// </summary>
		public double? Quan_cur {
            get { return getAttrVal<FDouble>("Quan_cur",null); }
            set { setAttrVal<FDouble>("Quan_cur", value); }
        }

        /// <summary>
        /// 金额
        /// </summary>
		public double? Amt_cur {
            get { return getAttrVal<FDouble>("Amt_cur",null); }
            set { setAttrVal<FDouble>("Amt_cur", value); }
        }

        /// <summary>
        /// 可用库存
        /// </summary>
		public double? Quan_stock {
            get { return getAttrVal<FDouble>("Quan_stock",null); }
            set { setAttrVal<FDouble>("Quan_stock", value); }
        }

        /// <summary>
        /// 药品剂型
        /// </summary>
		public string Id_dosage {
            get { return getAttrVal<string>("Id_dosage",null); }
            set { setAttrVal<string>("Id_dosage", value); }
        }

        /// <summary>
        /// 药品剂型编码
        /// </summary>
		public string Sd_dosage {
            get { return getAttrVal<string>("Sd_dosage",null); }
            set { setAttrVal<string>("Sd_dosage", value); }
        }

        /// <summary>
        /// 药理分类
        /// </summary>
		public string Id_pharm {
            get { return getAttrVal<string>("Id_pharm",null); }
            set { setAttrVal<string>("Id_pharm", value); }
        }

        /// <summary>
        /// 药理分类编码
        /// </summary>
		public string Sd_pharm {
            get { return getAttrVal<string>("Sd_pharm",null); }
            set { setAttrVal<string>("Sd_pharm", value); }
        }

        /// <summary>
        /// 毒麻分类
        /// </summary>
		public string Id_pois {
            get { return getAttrVal<string>("Id_pois",null); }
            set { setAttrVal<string>("Id_pois", value); }
        }

        /// <summary>
        /// 毒麻分类编码
        /// </summary>
		public string Sd_pois {
            get { return getAttrVal<string>("Sd_pois",null); }
            set { setAttrVal<string>("Sd_pois", value); }
        }

        /// <summary>
        /// 抗菌药分类
        /// </summary>
		public string Id_anti {
            get { return getAttrVal<string>("Id_anti",null); }
            set { setAttrVal<string>("Id_anti", value); }
        }

        /// <summary>
        /// 抗菌药分类编码
        /// </summary>
		public string Sd_anti {
            get { return getAttrVal<string>("Sd_anti",null); }
            set { setAttrVal<string>("Sd_anti", value); }
        }

        /// <summary>
        /// 价值分类
        /// </summary>
		public string Id_val {
            get { return getAttrVal<string>("Id_val",null); }
            set { setAttrVal<string>("Id_val", value); }
        }

        /// <summary>
        /// 价值分类编码
        /// </summary>
		public string Sd_val {
            get { return getAttrVal<string>("Sd_val",null); }
            set { setAttrVal<string>("Sd_val", value); }
        }

        /// <summary>
        /// OTC标识
        /// </summary>
		public bool? Fg_otc {
            get { return getAttrVal<FBoolean>("Fg_otc",null); }
            set { setAttrVal<FBoolean>("Fg_otc", value); }
        }

        /// <summary>
        /// 皮试标识
        /// </summary>
		public bool? Fg_skintest {
            get { return getAttrVal<FBoolean>("Fg_skintest",null); }
            set { setAttrVal<FBoolean>("Fg_skintest", value); }
        }

        /// <summary>
        /// 对应皮试服务
        /// </summary>
		public string Id_srvskin {
            get { return getAttrVal<string>("Id_srvskin",null); }
            set { setAttrVal<string>("Id_srvskin", value); }
        }

        /// <summary>
        /// 关联信息Map键值串
        /// </summary>
		public string Mapkeys {
            get { return getAttrVal<string>("Mapkeys",null); }
            set { setAttrVal<string>("Mapkeys", value); }
        }

        /// <summary>
        /// 服务物品关联信息MAP
        /// </summary>
		public FMap Mapinfo {
            get { return getAttrVal<FMap>("Mapinfo",null); }
            set { setAttrVal<FMap>("Mapinfo", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_emsdisrvmm", "Id_emsdi", "Id_emsdisrv", "Id_srv", "Id_mm", "Code_mm", "Name_mm", "Spec_mm", "Id_mmtp", "Sd_mmtp", "Name_mmtp", "Id_sup", "Name_sup", "Sd_roundmd", "Id_unit_base", "Name_unit_base", "Factor_mb", "Quan_num_base", "Quan_den_base", "Id_pgku_cur", "Name_pgku_cur", "Factor_cb", "Price", "Quan_cur", "Amt_cur", "Quan_stock", "Id_dosage", "Sd_dosage", "Id_pharm", "Sd_pharm", "Id_pois", "Sd_pois", "Id_anti", "Sd_anti", "Id_val", "Sd_val", "Fg_otc", "Fg_skintest", "Id_srvskin", "Mapkeys", "Mapinfo"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.emsdi.d.EmsDiSrvMmDTO";
        }
    }
}
